ddtrace/contrib/flask/patch.py

```diff
@@ -135,6 +137,9 @@ def _request_span_modifier(self, span, environ):
 
         span.set_tag_str(FLASK_VERSION, flask_version_str)
 
+        # set span.kind to the type of request being performed
+        span.set_tag_str(SPAN_KIND, SpanKind.SERVER)
```

Comment:
I don't think we need this is WSGI does this for us?

Reply:
Yup, I removed this